src/models/room.js

```diff
@@ -207,7 +207,12 @@ Room.prototype.getLiveTimeline = function() {
  */
 Room.prototype.resetLiveTimeline = function(backPaginationToken) {
     for (let i = 0; i < this._timelineSets.length; i++) {
-        this._timelineSets[i].resetLiveTimeline(backPaginationToken);
+        // Flush out non-live timelines. We do this to reduce the amount of memory
```

Comment:
this implies we're *only* flushing out the non-live timelines; I don't think that is correct?